impeller/renderer/context.cc

```diff
@@ -16,4 +17,37 @@ bool Context::UpdateOffscreenLayerPixelFormat(PixelFormat format) {
   return false;
 }
 
+void Context::GetTextureContents(const std::shared_ptr<Texture>& texture,
```

Comment:
We have something similar implemented in image_decode_impeller.cc, eventually we need to get rid of that and replace with our own logic to convert pixel formats. I'm not sure if the Skia implementation is unintentionally pulling in the software renderer.